config/admin/integrations/webhook-incoming/uptimerobot-incoming.js

```diff
@@ -0,0 +1,83 @@
+/**
+
+  This script is currently being used from https://github.com/freeCodeCamp/chat-config/tree/main/config/admin/integrations/
+
+   Attributions:
+     @license MIT
+     @version 0.2
+     @author  CrazyMax, https://github.com/crazy-max
+     @updated 2018-10-14
+     @link    https://github.com/crazy-max/rocketchat-uptimerobot
+ */
+
+/* globals console, _, s */
+
+const USERNAME = 'Uptime Robot';
+const AVATAR_URL =
+  'https://raw.githubusercontent.com/crazy-max/rocketchat-uptimerobot/master/res/avatar.png';
+
+const convertAlertDuration = (seconds) => {
+  let days = Math.floor(seconds / (3600 * 24));
+  seconds -= days * 3600 * 24;
+  let hours = Math.floor(seconds / 3600);
+  seconds -= hours * 3600;
+  let minutes = Math.floor(seconds / 60);
+  seconds -= minutes * 60;
+
+  let result = '';
+  if (days > 0) {
+    result += ' ' + days + ' days';
+  }
+  if (hours > 0) {
+    result += ' ' + hours + ' hours';
+  }
+  if (minutes > 0) {
+    result += ' ' + minutes + ' minutes';
+  }
+  if (seconds > 0) {
+    result += ' ' + seconds + ' seconds';
+  }
+
+  return result;
+};
+
+/* exported Script */
+class Script {
+  /**
+   * @params {object} request
+   */
+  process_incoming_request({ request }) {
+    let data = request.content;
+    let attachmentColor = `#A63636`;
+    let statusText = `DOWN`;
+    let isUp = data.alertType === `2`;
+    if (isUp) {
+      attachmentColor = `#36A64F`;
+      statusText = `UP`;
+    }
+
+    let attachmentText = `[${data.monitorFriendlyName}](https://uptimerobot.com/dashboard.php#${data.monitorID}) is ${statusText}`;
+
+    if (isUp) {
+      attachmentText += ` It was down for ${convertAlertDuration(
+        data.alertDuration
+      )}`;
+    } else {
+      attachmentText += ` Reason: ${data.alertDetails}`;
+    }
+
+    return {
+      content: {
+        username: USERNAME,
+        icon_url: AVATAR_URL,
+        text: `@here The uptime status for [${data.monitorFriendlyName}](https://uptimerobot.com/dashboard.php#${data.monitorID}) has changed.`,
+        attachments: [
+          {
+            text: attachmentText,
+            color: attachmentColor
```

Comment:
Just one quick question here. 

The `attachmentText` includes the `statusText` which has the relevant info for the notification. When working on the GitHub webhook integration, I discovered that the attachments field is length limited and will substring the content if it's too long.

Will that be a potential concern for not receiving the relevant information in the notification?

If not, then LGTM

Reply:
Interesting, I think we should modify this. Let me check that.